testing/file_test/file_test_base.cpp

```diff
@@ -405,6 +405,7 @@ static auto RunSingleTest(FileTestInfo& test, bool single_threaded,
     }
 
     if (single_threaded) {
+      llvm::errs() << "\nTEST: " << test.test_name << ' ';
```

Comment:
Maybe only print this when !FLAGS_dump_output? It's essentially a small variant of the execution mode, and will already be printing the test name (see line 388)

Reply:
Note, it might even make sense to merge with the 388 dump so that if a crash is related to ProcessTestFile, it'd be correctly associated

Reply:
Done! PTAL